addons/binding/org.openhab.binding.homie/README.md

```diff
@@ -0,0 +1,50 @@
+# Homie Binding
+
+This is the binding for devices that complie with the [Homie MQTT Convention](https://github.com/marvinroger/homie). This binding allows you to integrate all devices, as long as they complie with the specification.
+
+## Naming definition
+
+Homie term | Eclipse Smart Home term
+---------- | -----------------------
+Device     | Thing
+Node       | Channel
+
+## Discovery
+
+Discovery is done by browsing the MQTT topics located below the basetopic defined in the binding configuration. So you have to make sure all your Homie devices communicate using the same basetopic.
+
+## Additional convention
+
+If you want openHAB to render your nodes properly, you have to provide the following topics.
```

Comment:
Agreed. Check https://github.com/marvinroger/homie/issues/12#issuecomment-271895170 for an ongoing discussion and add whatever might be missing